.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,31 @@
-<!-- Describe your pull request here -->
+**Describe your pull request here**
 
 ----
 
 Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).
 
-* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title
-* New recipes not directly relevant to the biological sciences need to be submitted to [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda
-* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please add the `please review & merge` label.
+* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
+* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
+* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
 * Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.
 
 <details>
-  <summary>BiocondaBot commands</summary>
-      
+  <summary>Please make use of the following BiocondaBot commands whenever appropriate:</summary>
+
 Everyone has access to the following BiocondaBot commands, which can be given in a comment:
 
-  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
-  * `@BiocondaBot please add label` will add the `please review & merge` label.
-  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
+  * `@BiocondaBot please update`
```

Comment:
Putting this on a single line is just a convenience thing when wants to copy this.
(But the markdown file preview still shows it in one line... I might've to refine that then..)

Reply:
Okay, I just added `<br/>` tags for this.
The commit message has `visibility/copy-paste-ability` in it. The `copy-paste-ability` is true (but minor). But well, seeing it rendered, I'm not so sure about the `visibility` part since the gray-on-white differences don't add much visual distinction and I think I find the non-line-broken version more appealing.

Dammit, this is definitely bikeshedding territory :/ . Happy to leave it as it was before if someone else thinks this change does not provide much benefit!

Reply:
How about a table? Let me try it in this comment (not finished and also left out `@` because I don’t actually want to trigger the bot):
|utter this|to do this|
|-|-|
|BiocondaBot please update|merge the master branch into a PR|
|BiocondaBot please add label|add the `please review & merge` label

Hm, this seems to work quite well because I can now triple-click the cell to select the full command.

Reply:
Very good idea! Why didn't I think about that :D?

Reply:
With the shortened texts from https://github.com/bioconda/bioconda-recipes/commit/c0a5bc37987c787342cefd0cacbd751b03f4694c (which somehow does not show up in this PR for me right now!??) it looks like this:

----
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

|     |     |
| --- | --- |
| `@BiocondaBot please update` | Merge the master branch into a PR. |
| `@BiocondaBot please add label` | Add the `please review & merge` label. |
| `@BiocondaBot please fetch artifacts` | Post links to CI-built packages/containers. <br />You can use this to test packages locally. |

Comments from non-members that include `@bioconda/<team>` will be automatically reposted to notify the addressed team.

For members of the Bioconda project, the following command is also available:

|     |     |
| --- | --- |
| `@BiocondaBot please merge` | Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts. |

</details>

Reply:
The empty table heading looks weird, but tables apparently need headers. This doesn’t work:

|-|-|
|column 1|column 2|

[Stackoverflow question on this problem](https://stackoverflow.com/questions/17536216/)

But plain HTML works:

<table>
<td>hand-coded table</td><td>second column</td></tr>
<td><code>code</code></td><td>and the final cell</td></tr>
</table>

Reply:
I’ve made that change now.

Reply:
Yeah, I didn't find the empty header not terrible distracting, so didn't care ;).